### ISSUE TITLE
[SUGGESTION] Water blocks (for pool, aquarium, fountain, bathtub)

### ISSUE THREAD

**Issue by Uncle-Ulty** [tsunamayo/Starship-EVO#3442]
Hi,

I was building a fancy luxurious ship, and I thought: It would be nice to have a pool!

But the problem is, I wouldn't be able to propper use it, cuz we can't place water inside the ship!

The water feature already exists in the ring, and It would be nice to have this feature as a block, small or large.

The upper direction should be the one opposite to the base where you select to build the block. This feature would serve as a decoration for fountains, bathtubs, pools, aquariums etc.

BONUS idea:

If it's possible to implement the water block, maybe you could consider an irregular/not flat surface as a kind of waterfall? And maybe use gravity as a reference?

I'd love to finish my spaceship 5 stars hotel with several pools !

**Comment by Briaireous** [comment]
Water would be absolutely amazing, I wonder if you can use something like the shipyards sizable area of influence as a start

**Comment by Jorpis-Jorah** [comment]
Yes, I was thinking of suggesting it too! But some things I want to add:


- The water shouldn't have physics, because this could affect performance
- It should just be a block without collision to have the feel that you're swimming
- I like the water effect on the torus, so it should have similar look
- Colorable, so you could make it green (Pits of acid :))

**Comment by BigBadKangaroo** [comment]
That could also improve the spacestations that are comming in the future. I Like this idea
